### DISCUSSION TITLE
Beta申込時に入力した組織名を忘れてしまっていた

### DISCUSSION THREAD

**Discussion by netmarkjp** [topotal/waroom-roadmap#11] [Others]
Betaを申し込んだときの組織名の文字列を忘れてしまいinvitationを受けるときに悩みました。
申込受付メールなり、サービス開始メールなりに記載いただけると助かったと思います。

※結果として、勘でいくつか試したところ正解を引き当てました

**Comment by sawa-zen** [comment]
こちら https://github.com/topotal/waroom-roadmap/discussions/12 と同様にログイン時の組織名の入力でわからなくなってしまったということでしょうか？

やはり組織名の入力が必須になっていることで混乱を招いていると今回のリリースで明らかになったので、そもそも組織名入力を必要としないログインにできるよう改善できないか模索しつつ、メール側のテンプレートも改善しようと思います 👍🏻

**Comment by netmarkjp** [comment]
わかりづらくてすみません。
こちらは「入力した文字列を探そうにも手がかりがなくて悲しかった」ということをお伝えしたかったのでした 😇
